tests/release/test_release.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+from collections.abc import Callable
+from functools import partial
+from typing import Any
+
+import pandas as pd
+import pytest
+from _pytest.fixtures import FixtureRequest
+from vivarium_testing_utils import FuzzyChecker
+
+from pseudopeople.interface import (
+    generate_american_community_survey,
+    generate_current_population_survey,
+    generate_decennial_census,
+    generate_social_security,
+    generate_taxes_1040,
+    generate_taxes_w2_and_1099,
+    generate_women_infants_and_children,
+)
+from pseudopeople.noise_entities import NOISE_TYPES
+from pseudopeople.schema_entities import COLUMNS, DATASET_SCHEMAS
+from pseudopeople.utilities import (
+    count_number_of_tokens_per_string,
+    load_ocr_errors,
+    load_phonetic_errors,
+    load_qwerty_errors_data,
+)
+from tests.integration.conftest import IDX_COLS, _get_common_datasets, get_unnoised_data
+from tests.utilities import (
+    initialize_dataset_with_sample,
+    run_column_noising_tests,
+    run_omit_row_or_do_not_respond_tests,
+)
+
+DATASET_GENERATION_FUNCS: dict[str, Callable[..., Any]] = {
```

Comment:
Im pretty sure this constant exists in the integration(?) tests folder. Just make a constants file for tests like you did with the utilities file and put stuff like this there.